src/Test/WebDrapo.Test/Pages/Format.Test.html

```diff
@@ -5,7 +5,7 @@
 <body>
     <br />
     <span>Format</span>
-    <div d-datakey="dates" d-dataurlget="~/Data/GetDates"></div>
+    <div d-datakey="dates" d-dataurlget="~/Data/GetDates12Hour"></div>
```

Comment:
You have to change the file Format.html too. The Format.Test.html is generated from Format.html